recipe/meta.yaml

```
@@ -1,4 +1,4 @@
-{% set version = "1.6.1" %}
+{% set version = "1.6.2" %}
 {% set number = 1 %}
```

Need to reset build number here.
```suggestion
{% set number = 0 %}
```